lib/omise/schedule.rb

```diff
@@ -25,5 +25,9 @@ def occurrences
     def destroy(attributes = {})
       assign_attributes resource(attributes).delete
     end
+
+    def destroyed?
+      self.status == "deleted" ? true : false
```

Comment:
You don't need `self.`. This ain't Python

Reply:
Also `status == "deleted"` is good enough.

Reply:
@robinclart fixed krub, thanks a lot.